### DISCUSSION TITLE
Custom indentation depth

### DISCUSSION THREAD

**Discussion by Fricsion** [danny-avila/LibreChat#5111] [Feature Requests & Suggestions]
### What features would you like to see added?

Thanks for the great software!

I think having custom indentation depth level enhances usability significantly. 

Below is the example of me asking random question on iPhone. You can see there’s unnecessary nothing that I wish there weren’t. 
![IMG_3167](https://github.com/user-attachments/assets/535ed2f2-be01-4d40-bec4-92b674d90734)

### More details

That’s all. 

### Which components are impacted by your request?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This was done so triple digit numbered lists wouldn’t be cut off. I do think it should be closer together though as that is not the norm
